lib/collection/src/collection_manager/optimizers/segment_optimizer.rs

```diff
@@ -541,9 +541,14 @@ pub trait SegmentOptimizer {
                 .iter()
                 .all(|s| matches!(s, LockedSegment::Original(_)));
 
+        let num_points_to_optimize = optimizing_segments
```

Comment:
Here we count the number of points from the original segments. I wonder if it may be better to count the number of points in the final optimized segment. It would more closely reflect the number of points actually being optimized. Wdyt?

Reply:
> It would more closely reflect the number of points actually being optimized. Wdyt?

result of segment optimization is binary, right? i.e. either all points of a segment are optimized or none of them are optimized.

Reply:
No. Different segments having the same point ID will now be merged into one. That means the number of points in the final segment can be a bit less.